source/includes/_fields_specification.md

```diff
@@ -0,0 +1,490 @@
+
+## Annotate
+
+> mark the `unique_field` as the unique ID for the records
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "unique_field",
+        "annotation": "id"
+    }
+}
+```
+
+> mark the `category_field` as a facet
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "category_field",
+        "annotation": "facet"
+    }
+}
+```
+
+> make the `facetted_field` facet be sorted by descending count
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "facetted_field",
+        "annotation": "facetsort",
+        "args": ["-count"]
+    },
+}
+```
+
+> make the `datetime_field` have an hourly precision when used in timeseries
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "datetime_field",
+        "annotation": "timeseries_precision",
+        "args": ["hour"]
+    },
+}
+```
+
+> mark the `nondescript_number` field as representing square kilometers
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "nondescript_number",
+        "annotation": "unit",
+        "args":["km2"]
+    },
+}
+```
+
+> force the `decimal_field` precision to 5 digits after the decimal point
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "decimal_field",
+        "annotation": "decimals",
+        "args": [5]
+    },
+}
+```
+
+> mark the `multivalued_field` as multivalued, with a comma as the values separator
+
+```json
+{
+    "name": "annotate",
+    "args": {
+        "field": "multivalued_field",
+        "annotation": "multivalued",
+        "args": [","]
+    },
+}
+```
+
+Annotation are a mean to configure special behavior for the fields. Some annotations are only available for certain field types. Setting the facet annotation on a field unlocks other annotations for the field.
+
+Annotation name | Field type | Description
+--------------- | ---------- | ----------
+`id`|all field types | Whether this field should constitute one of the keys of the records unique IDs. If no field has this annotation, all fields contribute to the creation of the records unique ID.
+`facet` | `date`, `datetime`, `int`, `decimal`, `text` | Whether the field can serve as a filter
+`facetsort` | all fields type, facet only | How to sort the facets. Possible arguments are `count` and `-count` for all field types, `alphanum` and `-alphanum` for `date`, `datetime` and `text`, `num` and `-num` for `decimal` and `int`
+`disjunctive` | `decimal`, `int` and `text`, facet only | Whether multiple values can be selected for the facet
+`timeseries_precision` | `date` and `datetime` | precision of the field when used in a timeseries query. Possible arguments are `year`, `month` and `day` for `date`, `hour` and `minute` for `datetime` 
+`timerangeFilter` | `date` and `datetime`, facet only | Whether to active the timerange filter 
+`unit` | `int` and `decimal` | The unit of the field possible argument values are given [in the dedicated chapter](#units)
```

Comment:
Maybe split the description with a comma or in two sentences?

Reply:
:+1: